runtime/src/lib.rs

```diff
@@ -93,7 +148,7 @@ pub mod opaque {
 /// up by `pallet_aura` to implement `fn slot_duration()`.
 ///
 /// Change this to adjust the block time.
-pub const MILLISECS_PER_BLOCK: u64 = 60_000;
+pub const MILLISECS_PER_BLOCK: u64 = prod_or_fast!(15_000, 5_000);
```

Comment:
@nathanwhit can you confirm that changing from PoW with 60 secs block time to PoW -> PoS with 15 secs block time will not brick the chain ? 

Is the check relevant only for a PoS chain or is it relevant in general, even for PoW ?

Reply:
It won't brick the chain, yeah.

> Is the check relevant only for a PoS chain or is it relevant in general, even for PoW ?

Only for PoS

Reply:
This issue is only related to changing the epoch time when BABE is used for block authoring. The actual thing that cannot be changed is `EpochDuration` a constant set up in the BABE pallet config which does not exist in Creditcoin's PoW implementation.

After deploying BABE, changing the blocktime will brick the chain because `EpochDuration` is calculated by going from block time to minutes -> hours -> `EpochDuration`.
 
It does not brick the chain when switching from PoW to PoS and enabling BABE+GRANDPA.